api/views/schemas/group_memberships.py

```diff
@@ -10,8 +10,10 @@ class GroupMemberSchema(Schema):
 
     members_to_add = fields.List(fields.String(validate=validate.Length(equal=20)), required=True, load_only=True)
     owners_to_add = fields.List(fields.String(validate=validate.Length(equal=20)), required=True, load_only=True)
-    members_to_remove = fields.List(fields.String(validate=validate.Length(equal=20)), required=True, load_only=True)
-    owners_to_remove = fields.List(fields.String(validate=validate.Length(equal=20)), required=True, load_only=True)
```

Comment:
Looks like `required=True` was removed from `members_to_remove` and `owners_to_remove`, was that intentional?